src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -1723,11 +1756,38 @@ def _autoscaleModeUpdated(self):
 
         colormap = self.getColormap()
         if colormap is not None:
+            enableCentralPercentile = (
+                self._autoScaleCombo.currentText()
+                == _AutoscaleModeComboBox.DATA[Colormap.PERCENTILE][0]
+            )
             with self._colormapChange:
+                if enableCentralPercentile:
+                    colormap.setAutoscalePercentile(
+                        from_central_percentile_to_lateral_percentile_range(
+                            self._centralPercentileWidget.value()
+                        )
+                    )
                 colormap.setAutoscaleMode(mode)
 
         self._updateWidgetRange()
 
+    def _updateCentralPercentileVisibility(self):
```

Comment:
```suggestion
    def _updateCentralPercentileEnabled(self):
```
to match what it actually does